pkg/sfu/connectionquality/connectionstats.go

```diff
@@ -181,3 +215,60 @@ func ToAnalyticsVideoLayer(layer int, layerStats *buffer.LayerStats) *livekit.An
 		Frames:  layerStats.Frames,
 	}
 }
+
+func (cs *ConnectionStats) getBytesFramesFromStreams(streams []*livekit.AnalyticsStream) (totalBytes uint64, totalFrames uint32, maxLayer int32) {
```

Comment:
I think we should not do this function. I have not read `rtcscore` method. Is it expecting to mash all layers together? Different layers are at different frame rates. Is that going to cause an issue?

Is it enough to use the stats from max expected layer?

Reply:
@boks1971 this function calculates frames/bytes for each layer, and returns the values for the Max layer available. It does not mash up. rtcscore expects data for a given layer frame rate and bit rate.

Reply:
Sorry @shishirng . Read that wrong. Don't think we need to iterate. With the latest code, I have delta stats coming all the way from `RTPStats` module. There should be only one sample per interval. You can pretty much sort the slices by `layer.Layer` and take the last element values.

Also, realised that `layer.Layer` is coming from `AnalyticsStream` which is from protocol. Not sure if that is numbers 0, 1, 2 or the protocol quality. Don't remember. So, some of my comments here may be inaccurate.